README.md

```diff
@@ -1,24 +1,10 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+#Rails Type
+This app uses Rails 5
 
-Things you may want to cover:
+#What Does It Do?
 
-* Ruby version
+This app currently serves as a database for ST passengers. It allows for sorting, filtering, and searching passengers in the database. We’ve implemented a color code system so ST dispatchers can easily see which passengers are expired, about to expire, deactivated, or lack a needed doctor’s note. This app also has differing admin and non-admin views. A chron job runs every morning at 4:30am to deactivate expired passengers.
 
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+This app may later be expanded to serve other ST department needs, as well.
```

Comment:
Actually, the expansions they're talking about are comprehensive, require meetings, do not block completing the app for use as it is, and will in fact be done probably a few months from now. :)